doc/man5/flux-config-job-manager.rst

```diff
@@ -27,6 +27,17 @@ inactive-age-limit
 inactive-num-limit
    (optional) Integer maximum number of inactive jobs retained in the KVS.
 
+stop-queues-on-restart
```

Comment:
commit nit: messaage -> message

Reply:
Thanks! Force-pushed a fix and will set MWP.